packages/fungi/src/ContextChips/index.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import { IconButton } from "@leafygreen-ui/icon-button";
+import { RichLink } from "@lg-chat/rich-links";
```

Comment:
Oops I think I gave you bad info -- I believe we want to import from `Message.Links` https://github.com/mongodb/leafygreen-ui/tree/main/chat/message#messagelinks